homepage.html

```diff
@@ -228,9 +228,9 @@ <h3>This Month's Topics</h3>
                             {{presenter.name}} {% if not forloop.last %}, {% endif %}
                           {% endfor %}<br />
                       {% if topic.experience_level %}
-                      Experience Level: {{ topic.get_experience_level_display }}<br />
+                      BUTTS Experience Level: {{ topic.get_experience_level_display }}<br />
```

Comment:
Maybe remove BUTTS from here.

Reply:
There is a JavaScript function in the home page associated with trimming and toggling the description.  If we don't need it anymore could you delete it?

Reply:
MAAAYBE :D

Reply:
This is an extra homepage.html so I'm deleting it.  The real one is in chipy_org/templates/homepage.html.  Also removing the unneeded Javascript.